api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthorizationKeycloak.java

```diff
@@ -96,6 +98,25 @@ public void setDelegateToKafkaAcls(boolean delegateToKafkaAcls) {
         this.delegateToKafkaAcls = delegateToKafkaAcls;
     }
 
+    @Description("The time between two consecutive grants refresh runs in seconds. The default value is 60.")
+    public Integer getGrantsRefreshPeriodSeconds() {
```

Comment:
are you missing the `@Default` to 60 or it's defined somehow elsewhere?

Reply:
It will default to 60 in strimzi-kafka-oauth lib. You think it's better to let operator set the default? That indeed seems a pattern used as much as possible. I'll try fix that.

Reply:
You can keep it in the library as well. You don't have to remove it from there.

Reply:
Yes, didn't touch oauth lib.